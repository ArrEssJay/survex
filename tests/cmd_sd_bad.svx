; pos=fail error=7
*fix 1 reference 0 0 0 ; so there's some survey data

*sd tape metres
*sd tape 0.05 furlongs
*sd tape 0.05 furlongs per fortnight
*sd tripe 0.9 feet
*sd tape tripe 0.9 feet
*sd tape compass 0.4 meters
*sd 0.9 feet
