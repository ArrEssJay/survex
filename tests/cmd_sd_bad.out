./cmd_sd_bad.svx:4:16: error: Expecting numeric field, found "metres"
./cmd_sd_bad.svx:5:15: error: Unknown units "furlongs"
./cmd_sd_bad.svx:6:15: error: Unknown units "furlongs"
./cmd_sd_bad.svx:7:5: error: Unknown quantity "tripe"
./cmd_sd_bad.svx:8:15: error: Expecting numeric field, found "tripe"
./cmd_sd_bad.svx:9:22: error: Invalid units "meters" for quantity
./cmd_sd_bad.svx:10:5: error: Unknown quantity ""

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 1 survey station, joined by 0 legs.
There are 0 loops.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   1 0-node.
There were 0 warning(s) and 7 error(s) - no output files produced.
